### PR TITLE
Missing default build parameters for create release branch

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Review by evacchi — APPROVED** [review]

**Comment by radtriste** [comment]
@cristianonicolai This had an impact on the release ?

**Review by radtriste — APPROVED** [review]

**Comment by radtriste** [comment]
In fact `DISPLAY_NAME` and `RELEASE_BRANCH` are already set and others have default values... What was the failure there ?

**Comment by cristianonicolai** [comment]
@radtriste we were getting:
`NoSuchMethodError: No such DSL method 'addStringParam' found among steps`

File: Jenkinsfile.release [hunk 1]
```diff
@@ -94,7 +94,7 @@ pipeline {
                 script {
                     echo "Call ${CREATE_RELEASE_BRANCHES} job"
 
-                    def buildParams = [:]
+                    def buildParams = getDefaultBuildParams()
```
Comment (radtriste) [inline Jenkinsfile.release:97]: Oh I see now ...
Correction should just have been this one. No need to use `getDefaultBuildParams` and put all other parameters in the call.

```suggestion
                    def buildParams = []
```

@cristianonicolai Could be set like that after the release IMO ;)

**Pull request merged**